source/dashboard.rst

```diff
@@ -62,3 +62,48 @@ Clone Github repository
 -----------------------
 
 See :doc:`github`.
+
+Organizing your repositories with categories
+--------------------------------------------
+
+Over time, the list of repositories you deal with can grow long.  As it does, it can become difficult to
+find a repository you are looking for in the "Recent repositories" list when you have opened many other
+repositories since you opened the one you are looking for.  To help find repositories quickly, you can
+group your repositories into categories that will display in alphabetical order on the Dashboard.
+
+Create new category
+^^^^^^^^^^^^^^^^^^^
+
+While the Dashboard is being displayed (after application start-up), you create a new repository category
+by doing the following:
+
+- right click the repository you want to group into a category,
+- select "Categories",
+- select "Add new..." from the resulting fly-out menu,
```

Comment:
```suggestion
- select "Add new..." from the context menu,
```

Reply:
Hi, @RussKie !

Actually, only the "Categories" item is on the context menu.  Another fly-out menu is required to be accessed to be able to select the "Add new..." item.

Reply:
slightly more in style - used before in the doc
```suggestion
- select "Add new..." from the resulting submenu,
```

Thanks for the contribution. The doc need a lot more of care...